autocut/daemon.py

```diff
@@ -22,11 +22,12 @@ def run(self):
     def _iter(self):
         folder = self.args.inputs[0]
         files = sorted(list(glob.glob(os.path.join(folder, "*"))))
-        videos = [f for f in files if utils.is_video(f)]
+        media_files = [f for f in files if utils.is_video(f) or utils.is_audio(f)]
```

Comment:
make two list maybe better ?

audio_files = []
video_files = []

Reply:
Thanks for your suggestion, I have also considered it. However, since cut.py:Merger has not yet added support for audio files, processing audio in daemon mode will cause an error in the merge media step. I will choose an appropriate approach, a list or two lists, when resolving this issue.

Reply:
got it.